x-pack/test/functional/page_objects/index.ts

```diff
@@ -19,7 +19,6 @@ import { GraphPageProvider } from './graph_page';
 import { GrokDebuggerPageProvider } from './grok_debugger_page';
 // @ts-ignore not ts yet
 import { WatcherPageProvider } from './watcher_page';
-// @ts-ignore not ts yet
```

Comment:
Removing this `@ts-ignore` should have solved TypeScript errors I was getting locally, when getting the ReportingPageObjects with `getPageObjects(['reporting', ...])`

But I think those errors are still happening.